src/lizaalert/courses/models.py

```diff
@@ -455,3 +464,8 @@ class Meta:
 
     def __str__(self):
         return f"<Subscription: {self.id}, user: {self.user_id}, course: {self.course_id}>"
+
+    def update_status(self, status):
+        """Изменение статуса подписки на крус."""
+        self.status = status
+        self.save()
```

Comment:
лучше явно указать метод finish (чтобы разработчик не думал, какой статус надо установить)

```suggestion
    def finish(self, status):
        """Изменение статуса подписки на крус."""
        self.status = Subscription.Status.COMPLETED
        self.save()
```